template/content/app.js

```diff
@@ -0,0 +1,44 @@
+'use strict';
+
+var fetchConfig = require('zero-config');
+
+var ApplicationClients = require('./clients.js');
+var Endpoints = require('./endpoints.js');
+
+module.exports = Application;
+
+function Application(options) {
+    if (!(this instanceof Application)) {
+        return new Application(options);
+    }
+
+    var self = this;
+    options = options || {};
+
+    self.config = fetchConfig(__dirname, {
+        dcValue: 'todo',
+        seed: options.seedConfig,
+        loose: false
+    });
+
+    self.clients = ApplicationClients(self.config, {
+        logger: options.logger,
+        statsd: options.statsd
+    });
+    self.endpoints = Endpoints(self);
+
+    // Example data structure on application
+    self.exampleDb = {};
```

Comment:
I dont know whether the scaffold and the example app should be seperate.

I basically scaffolded out some simple endpoints to demonstrate how to do testing. Without those endpoints it's hard to demonstrate testing with just the health endpoint.

Previous scaffolds didn't have enough tests and left it open for developers to write their own tests. I want this scaffold to have some solid examples for how to write integration tests.